### DISCUSSION TITLE
Issues with connection

### DISCUSSION THREAD

**Discussion by MitziHai** [jlvaillant/intellicenter#31] [General]
So I was excited when I found this for home assistant….but I’ve installed and it won’t detect my intellicenter….any thoughts? It didn’t come up during auto discovery….I tried to add manually and input the IP, but no love there either.

**Comment by jlvaillant** [comment]
Hmmm… I don’t have a clear answer…. Could be a network related issue, the HA machine not able to see the IntelliCenter.
I can try to guide you with some basic tools you could use to investigate that on a Mac if you use one. Do you use VLANs on your network? When you use the IntelliCenter mobile app, does it see a “local” controller?

**Comment by MitziHai** [comment]
That would be great…I’m pretty technical, but this one has me stumped….No VLANs….and I don’t see in the mobile app where it says it’s a local controller or not, but it’s an wireless connection direct to my router (if that’s what you mean).

I do have a Mac (book) or an IPad whichever’s easier.

Thanks in advance for your help.

David

**Comment by jlvaillant** [comment]
First I would check that your IP is correct and accessible. Assuming X.X.X.X is your ip you should be able (from a terminal) to do:

ping X.X.X.X

assuming that works I would do:

nc X.X.X.X 6681

if if does connect try typing ‘ping’ (without the quotes + return). You should get a ‘pong’ back.

if this work, the integration should work with a manual IP. If not, please provide the log message you see.

Now, for the auto discovery, I use a free app (Discovery) available at https://apps.apple.com/us/app/discovery-dns-sd-browser/id1381004916?mt=12 <https://apps.apple.com/us/app/discovery-dns-sd-browser/id1381004916?mt=12>
When you launch it, your pentair system should appear under the _http._tcp.

I’m due to produce a patch that affects some “large” installations but the fact that it does not auto-discover is weird.

**Comment by MitziHai** [comment]
Ping works fine to the IP of the inteliconnect
No does not work (no connection)
Also it doesn’t appear in the auto discovery

**Comment by jlvaillant** [comment]
Sorry, I got to ask, are you sure you have an IntelliCenter unit?
Could you do a port scan on that IP address (network utility app unless you are running Monterey already)
